language-service/javascript/test/getGherkinFormattingEdits.test.ts

```diff
@@ -0,0 +1,30 @@
+import assert from 'assert'
+import { TextEdit } from 'vscode-languageserver-types'
+import { getGherkinFormattingEdits } from '../src'
+
+describe('getGherkinFormattingEdits', () => {
+  it('returns text edits that prettifies a Gherkin document', () => {
+    const gherkinSource = `Feature: Hello
+Scenario: World
```

Comment:
What about using [`reindent-template-literals`](https://gitlab.com/kao98/reindent-template-literals) to avoid breaking the indentation, as it is done with cucumber-js?

Reply:
The `gherkinSource` is intentionally unindented and the test verifies that it's indented after formatting.

Why do we need to worry about breaking indentation? (Not quite sure what you mean).

Reply:
I mean, the indentation in the test file itself is broken.

For example: https://github.com/cucumber/cucumber-js/commit/dd62cbb6a66c89a1bf8dc95ced5742173dee600f#diff-7e7cef54343b7de585c12b93987ac29f464c8ec6e7dcb33fea9b969e507c1073